src/vorta/borg/check.py

```diff
@@ -20,7 +22,9 @@ def finished_event(self, result: Dict[str, Any]):
         self.app.backup_finished_event.emit(result)
         self.result.emit(result)
         if result['returncode'] != 0:
-            self.app.backup_progress_event.emit(self.tr('Repo check failed. See logs for details.'))
+            self.app.backup_progress_event.emit(
+                trans_late('messages', f'Repo check failed. See <a href="file://{LOG_DIR}">logs</a> for details.')
```

Comment:
```suggestion
                trans_late('BorgCheckJob', f'Repo check failed. See <a href="file://{LOG_DIR}">logs</a> for details.')
```

Usually the context equals the class the string is used in. In this particular case there might be good reasons to use a different context, though I wouldn't consider `messages` a descriptive context.

Reply:
from the docs - `However sometimes you want to mark a string, use the english version and translate it later. In these cases you have to use trans_late and call translate on the string later.` I thought `translate it later` here meant that this function is used for strings which are not yet translated (untranslated), and translations for these strings will be done by people later, so I used that. But yes, now I understand that the "later" relates to the runtime 😄. I will replace these with translate, thanks.

Reply:
> I thought `translate it later` here meant that this function is used for strings which are not yet translated (untranslated), and translations for these strings will be done by people later, so I used that.

I see the source of misunderstanding. I'll think about a way to clarify that.

Reply:
What do you think about borgbase/vorta.borgbase.com#37?

Reply:
It's better. thanks :)